src/extra/ast.ts

```diff
@@ -613,13 +638,13 @@ export class ASTBuilder {
   visitFunctionExpression(node: FunctionExpression): void {
     var declaration = node.declaration;
     if (!declaration.arrowKind) {
-      if (declaration.name.text.length) {
+      if (declaration.ident.text.length) {
```

Comment:
Hmm, I'm not sure `ident` is good name. It could be misreading as `indent`. Why not keep the old `name`?

Reply:
What should the old `name` (that is still in use) become?

Reply:
I mean revert it to `declaration.name`

Reply:
> 
> 
> I mean revert it to `declaration.name`

Can't, `ident` is a getter function, `name` is still a property of DeclarationStatement

Reply:
Solved by discussion below

Reply:
I see. How about `id` of full name `identifier` instead `ident`?